docs/sdk/sdk-by-examples/simple-governance/bridging-it-all.md

```diff
@@ -0,0 +1,256 @@
+# Application - Bridging it all together
```

Comment:
can this be a shorter title? for the sidebar

Reply:
y